server/controllers/chartsController.js

```diff
@@ -60,10 +60,11 @@ const postProcessData = (data, studyTotals) => {
     const count = studySectionTotals.targetTotal
       ? studySectionTotals.targetTotal - studySectionTotals.count
       : 0
-    const percent = studyCountsToPercentage(
-      count,
-      studySectionTotals.targetTotal ?? studySectionTotals.count
-    )
+    const studySectionTargetValue =
+      studySectionTotals.targetTotal > studySectionTotals.count
+        ? studySectionTotals.targetTotal
+        : 0
```

Comment:
Don't we also want to keep the logic here that we should use the count if we don't have a target total?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/198/commits/e9480622bafa814cb0d48340a648e9c763dac837